tests/README.md

```diff
@@ -0,0 +1,82 @@
+# Testing
+
+## Overview
+
+The `/tests` directory contains HASH tests which for one reason or another aren't inlined with the code they correspond to (e.g. unit tests). This includes end-to-end tests, integration tests, and load tests.
+
+This `README` additionally includes helpful information about testing in HASH.
+
+## Debugging
+
+While using HASH as an end-user, various parts of the user interface -- specifically designed to help with development and debugging -- are hidden. As a developer, you can display these elements by running the following in your browser console:
+
+```js
+localStorage["hash.internal.debugging"] = "true";
+```
+
+## Tests
+
+### Backend Integration Tests
+
+Backend integration tests are located in the [`/tests/hash-backend-integration`](/tests/hash-backend-integration) folder.
+
+_The tests require a running instance of `hash-external-services`. see [the root README](/README.md#external-services-test-mode) for information on doing this without polluting the development database._
```

Comment:
All tests require to run external services, this is the main reason why they are here and makes them integration tests in the first place. (Some benchmarks requires that as well, this is why they are here as well)

Reply:
Perhaps you could help in rewriting this section for clarity.